commit 7188b0657880b3958bd44eac0d3add5220178252
Author: Tim Gardner <tim.gardner@canonical.com>
Date:   Thu Oct 18 12:30:20 2012 -0600

    UBUNTU: SAUCE: (no-up) fix highbank include path
    
    Continue to carry as its not hurting anything. Of course I could just
    build test without this patch, but thats a giant pain.
    
    Signed-off-by: Tim Gardner <tim.gardner@canonical.com>

diff --git a/arch/arm/mach-highbank/Makefile b/arch/arm/mach-highbank/Makefile
index 55840f414d3e..21e008d4f771 100644
--- a/arch/arm/mach-highbank/Makefile
+++ b/arch/arm/mach-highbank/Makefile
@@ -1,3 +1,5 @@
+KBUILD_CFLAGS += -I$(srctree)/arch/arm/mach-highbank/include
+
 obj-y					:= highbank.o system.o smc.o
 
 plus_sec := $(call as-instr,.arch_extension sec,+sec)
